NuRadioReco/framework/event.py

```diff
@@ -101,11 +103,51 @@ def get_stations(self):
     def set_station(self, station):
         self.__stations[station.get_id()] = station
 
+    def add_shower(self, shower):
```

Comment:
please add documentation to these functions. In particular what data should be stored in a shower object. The same for the following functions.